src/api/sci_api.cc

```diff
@@ -36,11 +36,162 @@
 #include "sparse_ci/sparse_ci_solver.h"
 #include "integrals/active_space_integrals.h"
 
+#include "sparse_ci/determinant.h"
+#include "sparse_ci/determinant_hashvector.h"
+#include "sparse_ci/sparse_state_vector.h"
+#include "sparse_ci/sparse_operator.h"
+#include "sparse_ci/sparse_fact_exp.h"
+#include "sparse_ci/sparse_exp.h"
+#include "sparse_ci/sparse_hamiltonian.h"
+
 namespace py = pybind11;
 using namespace pybind11::literals;
 
 namespace forte {
 
+/// Export the Determinant class
+void export_Determinant(py::module& m) {
+    py::class_<Determinant>(m, "Determinant")
+        .def(py::init<>())
+        .def(py::init<const Determinant&>())
+        .def(py::init<const std::vector<bool>&, const std::vector<bool>&>())
+        .def("get_alfa_bits", &Determinant::get_alfa_bits, "Get alpha bits")
+        .def("get_beta_bits", &Determinant::get_beta_bits, "Get beta bits")
+        .def("nbits", &Determinant::get_nbits)
+        .def("norb", &Determinant::norb)
```

Comment:
It isn't clear to me whether these are total bits or bits for just one spin case, spinorbitals or spatial orbitals.

Reply:
I documented the code and now added a new function nspinorb() that should probably deprecate nbits().